pandas/tests/indexes/test_base.py

```diff
@@ -1438,6 +1438,12 @@ def test_get_level_values(self):
         result = self.strIndex.get_level_values(0)
         tm.assert_index_equal(result, self.strIndex)
 
+        # GH 17414
```

Comment:
You didn't need necessarily remove your other note.  I was suggesting that you just add a reference to the issue somewhere in the comments.